src/components/dialog/demoBasicUsage/script.js

```diff
@@ -4,24 +4,32 @@ angular.module('dialogDemo1', ['ngMaterial'])
   $scope.alert = '';
 
   $scope.showAlert = function(ev) {
-    $mdDialog.show(
-      $mdDialog.alert()
```

Comment:
I will check. Meanwhile we should show a both approaches... I will do that with the merge.